pallets/duster/src/lib.rs

```diff
@@ -323,3 +323,24 @@ impl<T: Config> Contains<T::AccountId> for DusterWhitelist<T> {
 		AccountBlacklist::<T>::contains_key(t)
 	}
 }
+
+use hydradx_traits::pools::DustRemovalAccountWhitelist;
+
+impl<T: Config> DustRemovalAccountWhitelist<T::AccountId> for Pallet<T> {
+	type Error = DispatchResult;
```

Comment:
This should really be `type Result`, no?
Or the functions should return `Result<(), Self::Error>`, no?

Reply:
Yes, better.thanks